packages/eslint-plugin/package.json

```
@@ -52,7 +52,8 @@
     "typescript-eslint": "^7.5.0"
   },
   "peerDependencies": {
-    "eslint": ">=8.56.0"
+    "eslint": ">=8.56.0",
+    "typescript": "^5.0.0"
```

Can we make it optional? Not all of our users are using TypeScript:
```suggestion
    "typescript": "^5.0.0"
  },
  "peerDependenciesMeta": {
    "typescript": {
      "optional": true
    }
```

---

Yes, that's a good idea. It's optional in the @typescript-eslint package already.